### ISSUE TITLE
ttkDesigner: Protect MainWidget window to delete attempts

### ISSUE THREAD

**Issue by castben** [ceccopierangiolieugenio/pyTermTk#135]
Hi,
May I suggest to some how protect mainWidget to be deleted, or put a warning... or may be, delete it and create a new one that replaces it.
I've accidentaly hit delete on it and got following:

```
Traceback (most recent call last):
  File "/home/larry/.local/bin/ttkDesigner", line 8, in <module>
    sys.exit(main())
  File "/home/larry/.local/lib/python3.10/site-packages/ttkDesigner/app/main.py", line 47, in main
    root.mainloop()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 163, in mainloop
    self._mainLoop()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 172, in _mainLoop
    self._input.start()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/TTkTerm/input.py", line 77, in start
    self.key_process(stdinRead)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/TTkTerm/input.py", line 161, in key_process
    self.inputEvent.emit(kevt, mevt)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/signal.py", line 140, in emit
    slot(*args[sl], **kwargs)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 178, in _processInput
    self._key_event(kevt)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 239, in _key_event
    keyHandled = focusWidget.keyEvent(kevt)
  File "/home/larry/.local/lib/python3.10/site-packages/ttkDesigner/app/superobj/supercontrol.py", line 83, in keyEvent
    self._wid.parentWidget().removeSuperWidget(self._wid)
AttributeError: 'WindowEditorView' object has no attribute 'removeSuperWidget'. Did you mean: 'importSuperWidget'?
``` 

And my terminal hangs, I had to go to another one and kill it -- no big deal for me as I just messing around with it to get used to your new tool!

**Comment by ceccopierangiolieugenio** [comment]
The terminal become unresponsive because I disable CTRL-C in order to allow "Copy",
I need to check if I can reset the terminal in case of crash

I will remove the possibility to delete the main window soon.
Thanks

**Comment by castben** [comment]
Yes that make perfect sense. Thank you for your quick answer.

**Comment by ceccopierangiolieugenio** [comment]
Should be fixed in "**0.30.0a49**"
